### DISCUSSION TITLE
Radar Station does not exist.

### DISCUSSION THREAD

**Discussion by mwiikned** [weather-gov/api#365] [Q&A]
I used https://api.weather.gov/points/35.8344,-78.7818 to look up a location which should of been Raleigh / Durham, Raleigh-Durham International Airport (KRDU). However this gave the radarStation KRAX. KRAX from what I can tell is not listed in the https://api.weather.gov/stations list. Is this just a bug in the data and KRAX should be KRDU. Can locations have radarStations that do not exist in the Station list? Or am I mistaken that radarStations should always correspond to some station in that stations list.

**Comment by scadergit** [comment]
The "radarStation" property is the dopplar station that coveres that location. The /stations endpoint are observation stations (this has some ambiguity, sorry). If you follow the "ObservationStations" property in the /points lookup, you'll see a list of stations relevant to that location sorted by distance, with KRDU as the nearest. 

The next release of the API will include new radar endpoints if you are looking for dopplar stations.
